src/Presentational/BeneficiaryCard.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import "../index.css";
+
+const Tag = ({tags}) => (
+  <div>
+    {tags.map(tag => (
+      <div>{tag}</div>
+    ))}
+  </div>
+);
+
+export default class BeneficiaryCard extends Component {
+  render() {
+    return (
+      <div className="needCard"> 
+        <label className="needCard title">{this.props.name}</label>
+        <label className="needCard content">¿Dónde?</label>
+        <label className="needCard content">
+          <a href="{this.props.mapsurl}">{this.props.address}</a>
+        </label>
+        <label className="needCard content">¿Qué necesita?</label>
+
+        <Tag tags={["Alimentos", "Medicamentos", "Voluntarios"]} />
```

Comment:
Aquí el `Tag` element no tiene sentido porque estás imprimiendo varios tags. En todo caso sería un tag container aunque creo que es más limpio si haces algo como: 

`this.state.tags.map(tag => <Tag name={tag.name} />`